# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libid3tag/libid3tag-0.15.1b-add-pkgconfig-file.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN libid3tag-0.15.1b-orig/Makefile.am libid3tag-0.15.1b/Makefile.am
--- libid3tag-0.15.1b-orig/Makefile.am	2004-02-17 03:11:28.000000000 +0100
+++ libid3tag-0.15.1b/Makefile.am	2011-06-02 01:45:50.432004948 +0200
@@ -24,6 +24,10 @@
 SUBDIRS =		
 DIST_SUBDIRS =		msvc++
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = id3tag.pc
+DISTCLEANFILES = $(pkgconfig_DATA)
+
 lib_LTLIBRARIES =	libid3tag.la
 include_HEADERS =	id3tag.h
 
diff -ruN libid3tag-0.15.1b-orig/configure.ac libid3tag-0.15.1b/configure.ac
--- libid3tag-0.15.1b-orig/configure.ac	2004-01-24 00:22:46.000000000 +0100
+++ libid3tag-0.15.1b/configure.ac	2011-06-02 02:13:07.556004948 +0200
@@ -202,4 +202,4 @@
 
 AC_CONFIG_FILES([Makefile msvc++/Makefile  \
 	libid3tag.list])
-AC_OUTPUT
+AC_OUTPUT(id3tag.pc)
diff -ruN libid3tag-0.15.1b-orig/id3tag.pc.in libid3tag-0.15.1b/id3tag.pc.in
--- libid3tag-0.15.1b-orig/id3tag.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ libid3tag-0.15.1b/id3tag.pc.in	2011-06-02 01:47:06.300004947 +0200
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: id3tag
+Description: ID3 tag manipulation library
+Requires:
+Version: @VERSION@
+Libs: -L${libdir} -lid3tag
+Cflags: -I${includedir}
